package.json

```diff
@@ -50,7 +50,8 @@
     "node-sass": "^4.14.1",
     "nuxt-svg-loader": "^1.2.0",
     "prettier": "^1.19.1",
-    "sass-loader": "^8.0.2"
+    "sass-loader": "^10.1.1",
+    "stylelint": "^13.13.0"
```

Comment:
out of curiosity, why add back stylelint?

Reply:
Oh, I think that might have been an error. I'll remove it!